### DISCUSSION TITLE
How to transfer browser data from Windows 11 to Debian

### DISCUSSION THREAD

**Discussion by Rushikesh-Malave-175** [Alex313031/thorium#450] [Q&A]
Hello, quick question.

Recently I've started using Linux (Debian weekly build). I want to transfer the data of my thorium browser from Windows 11 to Debian. Can someone guide me to do this. By data I mean my sign ins, preferences, bookmarks. In short, I want to clone the exact same browser into Debian. (Don't really want to sign into all those accounts again :))

I do know how to install Thorium, but I do not seem to find how to transfer the browser data.

Thanks in advance

**Comment by haiderCho** [comment]
Thorium has Google Sync enabled, you can transfer everything using google sync. I'm Using Nobara, my Windows 11 and Linux is always synced.
Without Sync the process is messier and longer.
If you don't want to use sync, then export your Bookmarks in HTML file format, Export your Passwords in CSV format, then carry those files to Debian and import them on your Thorium. I tried copying Thorium User Data folder to Linux, It didn't work for me. Copying from Windows to Windows works but the sync gets paused so you have to login again on each profile anyway.
